user_guide/context/hooks.rst

```diff
@@ -125,12 +125,14 @@ class:
     /**
      * @BeforeSuite
      */
-    public static function prepare($scope)
+    public function prepare($scope)
```

Comment:
Well, this is a `@BeforeSuite` hook, so not making it static means it is a broken example. this should be changed to a different annotation for this first snippet